tests/test_trustme.py

```diff
@@ -59,6 +59,12 @@ def test_basics():
     assert hostnames == [u"test-1.example.org", u"test-2.example.org"]
 
 
+def test_intermediate():
+    ca = CA()
+    ca = ca.create_child_ca()
+    ca.issue_server_cert(u"test-host.example.org")
```

Comment:
This test feels a bit trivial... maybe that's OK, it's just a smoke test? But maybe we should at least check a few basic attributes, like that `server_cert`'s chain has two entries, the second one matches the child ca's chain, that sort of thing?